tests/integration_tests/chat_completion_suites/vision.py

```diff
@@ -0,0 +1,30 @@
+from aidial_adapter_openai.constant import ChatCompletionDeploymentType
+from tests.integration_tests.base import TestSuite, include_deployments
+from tests.integration_tests.constants import SAMPLE_DOG_RESOURCE
+from tests.utils.openai import user_with_attachment_url, user_with_image_url
+
+
+@include_deployments(
+    [
+        ChatCompletionDeploymentType.GPT4O,
+        ChatCompletionDeploymentType.GPT4O_MINI,
+        ChatCompletionDeploymentType.GPT4_VISION,
+    ]
+)
+def vision_common(s: TestSuite) -> None:
+    s.test_case(
+        name="image_in_content_parts",
+        messages=[
+            user_with_image_url("What is this image?", SAMPLE_DOG_RESOURCE),
```

Comment:
Maybe worth querying `Describe the image in one word` to save completion tokens.

Reply:
Good idea, done

Reply:
`max_tokens=800` is a bit too much for one word?

Reply:
Yeah, forgot that max_tokens do not include prompt tokens

Reduced to `max_tokens=5`